2-mandatory/1-weather-report.js

```diff
@@ -12,7 +12,12 @@
 */
 
```

Comment:
This looks very good 👍 
There might be another array method that you could use here instead of forEach - that will make things even easier.
Have a look at the `map` array method if you get some time.